hack/jenkins/e2e.sh

```diff
@@ -164,6 +166,7 @@ GCE_SLOW_TESTS=(
 
 # Tests which are not able to be run in parallel.
 GCE_PARALLEL_SKIP_TESTS=(
+    "GCE\sL7\sLoadBalancer\sController"
```

Comment:
Oh, and to be clear, PARALLEL_SKIP_TESTS is for tests that cannot run in parallel with *other* tests (e.g. tests that reboot nodes, resize clusters, saturate clusters with pods etc), because they break other tests.  Perhaps you want to update the comment accordingly.